samples/net/gptp/boards/frdm_k64f.conf

```diff
@@ -0,0 +1,9 @@
+# MCUX driver settings
+CONFIG_ETH_MCUX=y
+CONFIG_PTP_CLOCK_MCUX=y
+
+# Optionally you can use fixed MAC address
```

Comment:
What's the reason for this? I suggest removing and relying on stable unique MAC address which is now default.